setup.py

```diff
@@ -61,10 +61,11 @@ def extract_requirements(filename):
         "License :: OSI Approved :: Apache Software License",
         "Operating System :: OS Independent",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
```

Comment:
could we add python 3.9 here also?  It is the latest stable

Reply:
Shall we also update .travis.yml for all these supported Python 3.x version?

Reply:
Thank you!